_posts/2023-11-06-rebooting-the-diversity-equity-and-inclusion-working-group-in-sustain.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Rebooting the Diversity, Equity, and Inclusion Working Group in Sustain OSS"
+permalink: /blog/:title/
+---
+
+
+The Sustain OSS Diversity Equity and Inclusion working group was [relaunched on the 27th of October, 2023](https://discourse.sustainoss.org/t/dei-in-sustain-understanding-the-universal-perspectives/1310), and is calling for participation. [Join us](https://sustainoss.org/working-groups/diversity-equity-inclusion/) to:
+
+* Learn and share insights on diversity, equity, and inclusion.
+* Collaborate to promote long-term diversity, equity, and inclusion in open source.
+* Bridge the awareness gap in existing DEI programs and initiatives.
+
+
+## The Why
+
+The Sustain OSS Diversity, Equity, and Inclusion working group is open to individuals from diverse backgrounds with experience and interest in DEI within open source.
+The aim of the DEI Working Group at Sustain OSS is to promote and advance diversity, equity, and inclusion within open-source communities. We strive to create a more inclusive and equitable open-source ecosystem by:
+
+* Supporting communities in implementing DEI efforts and connecting them with experienced leaders in the open source.
+* Leading research initiatives to better understand how DEI impacts the community, contributors, users, and maintainers in open source.
+* Fostering ongoing discussions that promote the sustainability and adoption of field-tested DEI principles and best practices through events, meetups, and podcasts.
+* Collaborating with DEI champions to create and curate field-tested resources, reference guides, and industry benchmarks.
+
+Our efforts are dedicated to establishing DEI as an integral part of sustaining open source communities.
+
+## How can we achieve them?
+
+Our approach is rooted in collaboration and shared insights. We understand that diversity, equity, and inclusion aren't one-size-fits-all ideas. We intend to bring together enthusiasts, contributors, and leaders in open source by establishing a platform for meaningful conversations and problem-solving.
+We plan to achieve this by:
+
+* Hosting events, Podcast, workshops, and meetups.
+* Creating and curating reference guides, articles, and resources.
+* Providing support to open source community leaders through DEI office hours.
+
+## How to participate
+
+You can get involved and connect with like-minded individuals by:
+
+* Joining Our Community: Send an email to our [Google group](https://groups.google.com/g/sustain-dei-wg) to stay updated on our activities. Or join our discussions on the [Sustain OSS Discourse](https://discourse.sustainoss.org/t/dei-in-sustain-understanding-the-universal-perspectives/1310).
```

Comment:
I don't think this will work - you can't send an email to a Google group you're not a member of. I would change the email to suggest that they request to join the Google Group, or post in the Discourse or message you somehow (you can put your email or your LinkedIn or something).